android/app/src/test/java/com/zulipmobile/notifications/FcmMessageTest.kt

```diff
@@ -142,11 +142,6 @@ class MessageFcmMessageTest : FcmMessageTestBase() {
         )
     }
 
-    @Test
-    fun `optional fields missing cause no error`() {
-        expect.that(parse(Example.pm.minus("user_id")).identity.userId).isNull()
```

Comment:
This should turn into a case where we test that required fields are required. See 9620629a7571c7d981c54384cfe9f3d72c9fadbc or other recent changes in `git log --stat -p android/app/src/test/`.